index.html

```diff
@@ -87,6 +43,18 @@ <h2 class="title">
    <h6 class="text-header">
     Packages
    </h6>
+   <a class="card" href="" id="placeholder_card" style="display: none;">
+    placeholder_name
+    <span>
+    </span>
+    <span class="version">
+     placehholder_version
+    </span>
+    <br/>
+    <span class="description">
+     placeholder_description
+    </span>
+   </a>
```

Comment:
Even if this anchor is not visible, it is still present in the HTML of the page, right ?
If so, I don't think it follows PEP 503 specifications ? @pabnas sorry for being so picky, but I think it's really important to make sure we are PEP-compliant ^^

Reply:
yeah don't worry for that, this placeholder_card acts like a template for generating the other cards, i add it because the register action takes into account that exist an anchor to copy, when i run the update_pkgs.py the script removes all the existing packages leaving no anchor to copy this also happens when you clone and remove the existing packages.

One way to fix this is creating the structure in the actions.py

Reply:
Agree, I think it's better to put the structure in the .py file, so we don't have "dangling" anchors in the `index` file !